packages/api/ecs/async-operation/README.md

```diff
@@ -21,9 +21,13 @@ The built image is deployed to
 
 Logs will be output to `${stackName}-${OperationName}EcsLogs`
 
+## Development
+If you are making changes to this section of the repo, including security patches, you will need to follow the steps to
+build the Docker image, push the image to ECR, update the [example deployment](../../../../example/cumulus-tf/), and run your changes in Bamboo.
+
 ## Building and pushing Docker images
 
-For the following commands, replace `<build-number>` with the next build number. You can
+This step should be done as part of the release process. For the following commands, replace `<build-number>` with the next build number. You can
```

Comment:
We should probably consider if we want the ECR tags to be out of sync with the dockerhub release #s. 

If we're only publishing on release, we will have missing numbers from the dockerhub tags.... which is fine, but might lead to confusion.   Something to think about.

Reply:
I _think_ it's possible to keep the numbers in sync. I built and pushed a `53` to dockerhub. I also pushed `53` up to ECR, but that's because I didn't know it would have been done already for dev and test, and as you pointed out, I overwrote the `53` images you had already pushed up originally. So even if I didn't push to ECR, the numbers should have been in sync

Reply:
The concern is more like "Ticket 1: up to v54, merge.   Ticket 2: up to v55, merge.   Release: publish v 56 to ECR and dockerhub"

Reply:
I can add more text about trying to keep the build numbers in sync if you think that would be helpful.

Reply:
Is this something that has happened before? If so, how did we handle it?
Given that scenario, what I would think we'd want to do is to say, look at the latest version in ECR (assuming that pushing there for testing is done properly), and match the latest version in dockerhub to match. Does that seem ok? Or should we consult with the team? If that seems alright then I can add text somewhere to that effect...